modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring route admission policy
+
+By default, {microshift-short} prevents routes in multiple namespaces from using the same hostname. To accommodate different use cases, you can use the configuration option below to disable the `namespaceOwnership` check. This allows routes to claim different paths of the same hostname across namespaces.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameter in the `ingress.routeAdmissionPolicy.namespaceOwnership` field to `InterNamespaceAllowed` as shown in the following example to disable the check:
++
+.Example `config.yaml` route admission policy settings
+[source,yaml]
+----
+#....
+ingress:
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed <1>
+# ....
+----
+<1> Default value. Allows routes to claim different paths of the same host name across namespaces.
+To enable the checks, you can update the parameter in the `ingress.routeAdmissionPolicy.namespaceOwnership` field to `Strict`.This will prevent routes in different namespaces
+to claim the same host.
++
+[TIP]
+====
+If you complete all the configurations that you need to make in the {microshift-short} `config.yaml` file at the same time, you can minimize system restarts.
+====
+
+. Restart the {microshift-short} service by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart microshift
+----
```

Comment:
Is there a simple Verification step we should add somewhere?

Reply:
@pacevedom we had discussed to omit verification. Should I add this or not? Please suggest

Reply:
We removed them from the other PR. MicroShift does show the current config when it starts anyway, and we have the `show config` in nearby docs. Sorry for the noise, disregard.